microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,140 @@
+
+Similarly, in the **Performance** tool; when you import a performance trace recorded earlier, it needs to have been recorded on the same site and same configuration to be able to map from the performance report to the JavaScript that was running in the browser.
+
+An enhanced trace retains more information about the page. For example, an enhanced trace contains the execution context, the list of parsed scripts, and can optionally contain console messages, script sources, and a snapshot of the DOM tree.
+
+When a normal trace is imported in DevTools, the recorded data is used to populate the **Performance** or **Memory** tool. When an enhanced trace is imported in DevTools, a new DevTools window appears. This new window isn't connected to the web page that's running in your browser, and instead re-creates the environment in which the trace was originally recorded. For example, if a snapshot of the DOM was recorded in the enhanced trace, the **Elements** tool displays this snapshot. If console messages were recorded, the **Console** tool prints these messages. The **Sources** tool displays the scripts that were present during the recording.
+
+Using enhanced traces makes it possible to reliably resolve source code references found in imported traces to the actual runtime code in the **Sources** tool. Additionally, if source maps were presents when a trace was recorded, it will also be possible to resolve code references to their original source code.
+
+Enhanced traces can only be imported in Microsoft Edge, while normal traces are compatible with other browsers based on Chromium.
+
+
+<!-- ====================================================================== -->
+## Set the default trace type
+
+By default, the **Performance** and **Memory** tools export data as normal traces, but it's possible to choose which type of traces are export.
+
+To change the trace type:
+
+1. Open DevTools by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
+
+1. In DevTools, in the upper right, click **Settings** (![Settings button.](../media/settings-button-icon.msft.png)).
+
+1. Under **Persistence**, if you want to record enhanced traces, check **Export Enhanced Performance and Memory Traces**. Otherwise clear the option.
+
+    ![The Settings panel in DevTools, with the enhanced traces option checked](./share-traces-images/trace-type-setting.png)
+
+1. Enhanced traces can also optionally include console messages, script sources and DOM elements too. Choose what to include in enhanced traces by using the **Include console message in enhanced traces**, **Include script sources in enhanced traces**, and **Include DOM snapshots in enhanced traces (experimental)** options:
+
+    ![The Settings panel in DevTools, showing the other enhanced trace options](./share-traces-images/enhanced-trace-options.png)
```

Comment:
```suggestion
   ![The Settings panel in DevTools, showing the other enhanced trace options](./share-traces-images/enhanced-trace-options.png)
```
more or fewer leading spaces, consistent in each list, for right numbering result / indents